### PR TITLE
Add trace ID to clipboard debug info and title bar

### LINKED ISSUE #9146: Include full trace ID in "copy to debug info" and add short version to the ert title bar

### PR DESCRIPTION
**Pull request by HakonSohoel**
**Issue**
Resolves #9146

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'pytest tests/ert/unit_tests -n logical -m "not integration_test"'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Comment by HakonSohoel** [comment]
Shows the first 8 trace ID digits in the title bar: 

image

**Comment by andreas-el** [comment]
Possible to add a small test for this? There is already a test that checks some of the output from the clipboard button.
You could easily add something small there perhaps?

File: src/ert/trace.py [hunk 1]
```diff
@@ -9,3 +9,7 @@
 trace.set_tracer_provider(tracer_provider)
 
 tracer = trace.get_tracer("ert.main")
+
+
+def getTraceID() -> str:
```
Comment (berland) [inline src/ert/trace.py:14]: rename to  `get_trace_id()` for consistent style?

Comment (HakonSohoel) [reply src/ert/trace.py]: 👌

**Review by berland — APPROVED** [review]
LGTM!

**Pull request merged**